python/cuml/test/test_array.py

```diff
@@ -57,9 +57,7 @@
 test_dtypes_all = [
     np.float16, np.float32, np.float64,
     np.int8, np.int16, np.int32, np.int64,
-    np.uint8, np.uint16, np.uint32, np.uint64,
-    "float", "float32", "double", "float64",
-    "int8", "short", "int16", "int", "int32", "long", "int64",
+    np.uint8, np.uint16, np.uint32, np.uint64
```

Comment:
The old types were redundant right?

Reply:
yeah, the NumPy dtype (and CuPy for that matter) objects allow you to pass strings for the dtype, but that's part of the NumPy/CuPy codebase, not something we need to test on our end